docs/cloud/dev/spryker-cloud-commerce-os/create-and-restore-database-backups.md

```diff
@@ -126,6 +126,55 @@ It may take a few minutes for the DB to restore.
 {% endinfo_block %}
 
 
+### Restore a DB from AWS Backup
+
+1. In the AWS Management Console, go to **Services** > [**AWS Backup**](https://console.aws.amazon.com/backup).
+
+2. In the navigation pane, select **Backup vaults**.
+
+3. Click the backup vault `{environment-name}_rds_backup`.
+
+4. In **Recovery points** tab, click on the link start with _"continuous:db"_.
+
+5. Click on **Restore** on the right top of the screen.
+
+6. In **Restore to point in time** tab, choose _**Latest restorable time**_ or _**Specify date and time**_.
+
+7. In **Instance specifications** tab, set _**DB instance class**_ field and keep the rest of fields as default.
+
+8. In **Settings** tab, set _**DB Instance Identifier**_ as new DB name.
+
+9. In **Network and security** tab, set _**Virtual Private Cloud (VPC)**_ as RDS belongs to. Then select __**Subnet group**__ and _**Public accessibility**_.
```

Comment:
Do you mean that I should select the _**Virtual Private Cloud (VPC)**_ to which my RDS belongs? How do I know that?

Reply:
yes, name prefix shared on new commit.